test/test_dynamic_shapes.py

```diff
@@ -975,27 +997,42 @@ def get_sym_inp(inp):
             else:
                 return torch.SymFloat(to_node(seed_node, inp))
 
+        if fn == "float_pow":
+            if inp1 < 0:
+                return
+
+        if fn == "pow_by_natural":
+            if isinstance(inp1, float) or isinstance(inp2, float):
+                return
+            if inp2 < 0:
+                return
```

Comment:
This is a hack because I couldn't figure out how the regular xfail machinery was supposed to work

Reply:
@albanD, you wrote the original testing code, curious if you have an opinion